app/notify_admin.go

```diff
@@ -236,8 +295,24 @@ func (a *App) groupNotifyAdminByUser(data []*model.NotifyAdminData) map[string][
 func (a *App) groupNotifyAdminByFeature(data []*model.NotifyAdminData) map[model.MattermostPaidFeature][]*model.NotifyAdminData {
 	myMap := make(map[model.MattermostPaidFeature][]*model.NotifyAdminData)
 	for _, d := range data {
+		if strings.HasPrefix(string(d.RequiredFeature), string(model.WorkTemplates)) {
+			continue
+		}
 		myMap[d.RequiredFeature] = append(myMap[d.RequiredFeature], d)
 	}
 
 	return myMap
 }
+
+func (a *App) groupNotifyAdminByPlugin(data []*model.NotifyAdminData) map[string][]*model.NotifyAdminData {
+	myMap := make(map[string][]*model.NotifyAdminData)
+	for _, d := range data {
+		if strings.HasPrefix(string(d.RequiredFeature), string(model.WorkTemplates)) {
+			plans := strings.Split(d.RequiredPlan, ",")
```

Comment:
Looks like plugin notify admin requests store a comma separated list of plugins, however many the template needs to install. This will be variable as I think we'll create new templates for different use cases over time. So we'll want to make sure that the column that stores the plugin list can store all that info. Right now, I think it will be too short for some cases. The current schema for this table is:

```
     Column      |          Type          | Collation | Nullable | Default
-----------------+------------------------+-----------+----------+---------
 userid          | character varying(26)  |           | not null |
 createat        | bigint                 |           |          |
 requiredplan    | character varying(26)  |           | not null |
 requiredfeature | character varying(100) |           | not null |
 trial           | boolean                |           | not null |
```

In another part of the code, I suggest making requiredplan be work templates, and required feature the list of plugins to match the current notion of requiredplan as category, requiredfeature as subcategory. So if we do that, we'd want to also alter requiredfeature to be able to comfortably fix the maximum expected number of plugins a template can have. Not sure if users or third parties will ever be able to define their own work templates, but if so, it could be arbitrarily many plugins. We might want to change this to a `TEXT` column to accomodate.